packages/system-tests/scenarios/scaffolding.test.ts

```diff
@@ -219,4 +219,36 @@ describe('Scaffolding commands', () => {
     const elCount = await common.getQuickOpenElements();
     expect(elCount).to.equal(2);
   });
+
+  it('Should create Lightning interface', async () => {
+    // Invoke SFDX: Create Lightning interface command by name
+    await app.command('workbench.action.quickOpen');
+    await common.type('>SFDX: Create Lightning Interface');
+    await app.client.keys(['NULL', 'Enter', 'NULL'], false);
+    await app.wait();
+
+    const fileName = `lightningIntf_${new Date().getTime()}`;
+
+    // Enter file name
+    await common.type(fileName);
+    await app.client.keys(['NULL', 'Enter', 'NULL'], false);
+    await app.wait();
+
+    // Enter desired location (without slashes so it's OS-independent)
+    await common.type('force-appmaindefaultaura');
+    await app.client.keys(['NULL', 'Enter', 'NULL'], false);
+    await app.wait();
+
+    // Check that the new lightning evt is opened in a new tab
```

Comment:
Copy-paste error. 
// Check that the new lightning evt <-- should be lightning intf